etc/viam-csi-jetson-arm64.yml

```diff
@@ -20,10 +20,10 @@ AppDir:
   files:
     include:
     - /usr/local/lib/libviamsdk.so.noabi
+    - /lib/aarch64-linux-gnu/libpthread*
```

Comment:
Had to add `pthread` include explicitly in bundle when moving from focal to jammy.